src/referentielijsten/api/admin.py

```diff
@@ -82,3 +86,14 @@ def is_geldig(self, obj):
             return True
 
         return False
+
+    @admin.display(description=_("Actions"))
+    def items_link(self, obj):
+        url = (
+            reverse("admin:api_item_changelist")
```

Comment:
In django 5.2 you could use `reverse(..., query={...})` here, but Referentielijsten is still on django 4.2. Could you add a comment here to modify this once we're on 5.2? https://docs.djangoproject.com/en/5.2/ref/urlresolvers/

`# TODO replace with reverse(..., query=...) once Django is upgraded to 5.2`